internal/uvm/scsi/manager.go

```diff
@@ -67,6 +67,14 @@ type MountConfig struct {
 	Partition uint64
 	Encrypted bool
 	Options   []string
+	// EnsureFilesystem indicates to format the mount as `Filesystem`
+	// if it is not already formatted with that fs type.
+	// This is only supported for LCOW.
```

Comment:
If we're going to add this comment here for the new fields, we should probably add it for ther others as well.

Reply:
Added comments here. Let me know if they look good.